spec/system/candidate_interface/candidate_adding_referees_spec.rb

```diff
@@ -28,6 +29,24 @@
     i_see_referees_is_complete
   end
 
+  scenario 'Candidate enters an invalid email' do
```

Comment:
Perhaps we can remove this scenario. We have a unit test for it, and the accepatance tests work best if there's a single scenario per file (otherwise they become long and hard to read).

Reply:
Removed